include/battle.h

```diff
@@ -851,11 +851,25 @@ STATIC_ASSERT(sizeof(((struct BattleStruct *)0)->palaceFlags) * 8 >= MAX_BATTLER
 #define F_DYNAMIC_TYPE_IGNORE_PHYSICALITY  (1 << 6) // If set, the dynamic type's physicality won't be used for certain move effects.
 #define F_DYNAMIC_TYPE_SET                 (1 << 7) // Set for all dynamic types to distinguish a dynamic type of Normal (0) from no dynamic type.
 
-#define IS_MOVE_PHYSICAL(move) (GetBattleMoveCategory(move) == DAMAGE_CATEGORY_PHYSICAL)
-#define IS_MOVE_SPECIAL(move) (GetBattleMoveCategory(move) == DAMAGE_CATEGORY_SPECIAL)
-#define IS_MOVE_STATUS(move) (GetMoveCategory(move) == DAMAGE_CATEGORY_STATUS)
+static inline bool32 IsBattleMovePhysical(u32 move)
```

Comment:
were you not able to move those to move.h?

Reply:
Did not think to do it. Let me do it real quick

Reply:
yeah, I cannot, it causes a dependency loop. "BattleMoveX" should probably stay as battle functions.